### PR TITLE
[release-1.7] 🌱 Bump to Go 1.19.13

### PR DESCRIPTION
**Pull request by zhanggbj**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by zhanggbj** [comment]
pending on docker golang image

```
$ docker pull docker.io/library/golang:1.19.13
Error response from daemon: manifest for golang:1.19.13 not found: manifest unknown: manifest unknown
```

File: Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Build the manager binary
-ARG GOLANG_VERSION=golang:1.19.3
```
Comment (chrischdi) [inline Dockerfile:18]: Looks like 1.19.8 is the last version here :-(

Comment (zhanggbj) [reply Dockerfile:18]: docker.io got golang image for 1.19.12 :-) perhaps we could wait for 1 or 2 days for 1.19.13

Comment (sbueringer) [reply Dockerfile:18]: I would say let's wait a few days if it shows up. If it doesn't we can start asking around

Comment (chrischdi) [reply Dockerfile:18]: Ah, my bad. did wrongly look it up 👍

Comment (sbueringer) [reply Dockerfile:18]: xref https://github.com/docker-library/golang/issues/488

**Pull request merged**